src/extensions/default/InlineColorEditor/ColorEditor.js

```diff
@@ -172,7 +172,10 @@ define(function (require, exports, module) {
         // Update gradients in color square & opacity slider
         this.$selectionBase.css("background-color", colorObject.toHexString());
         this.$opacityGradient.css("background-image", "-webkit-gradient(linear, 0% 0%, 0% 100%, from(" + hueColor + "), to(transparent))");
-        
+        this.$opacityGradient.css("background-image", "-moz-linear-gradient(top, " + hueColor +" 0%, rgba(0,0,0,0) 100%");
+        this.$opacityGradient.css("background-image", "-o-linear-gradient(top, " + hueColor +" 0%, rgba(0,0,0,0) 100%");
+        this.$opacityGradient.css("background-image", "-ms-linear-gradient(top, " + hueColor +" 0%, rgba(0,0,0,0) 100%");
```

Comment:
CEF 2721 supports `-webkit-gradient`, `-webkit-linear-gradient` and `linear-gradient`. Which one, then, will be applied here?